### ISSUE TITLE
Allow wrapper package versions to include a pre-release postfix

### ISSUE THREAD

**Issue by rbabayoff** [meteor/meteor#6499]
Hey,

When trying to use a wrapper package pre release version, such as `0.0.1_2-rc1`, I get this:

```
=> Errors while setting up tests:

While reading package from `/home/vagrant/packages/wrapper-package`:
error: The package version 0.0.1_2-rc1 (specified with Package.describe) is not a valid Meteor package version.
Valid package versions are semver (see http://semver.org/), optionally followed by '_' and an integer greater or equal to 1.
```

Wrapper package versions need to support pre-release postfixes. For example, our practicalmeteor:mocha wrapper package is now at 2.1.0_7 and for meteor 1.3 rc1, we want to use 2.1.0_8-rc1, but we can't.

@tmeasday any chance this can get into 1.3? We can work around it easily, if not, by using 2.1.1-rc1.

**Comment by rbabayoff** [comment]
If I use 2.1.0-rc.1 when I publish the package, I don't get it, even when specifying a version constraint on it, I always get 2.1.0_7.

**Comment by tmeasday** [comment]
Yeah, I ran into this the other day. It's frustrating.

I can't see a change like this to the package system getting into 1.3 so we'll need to work around it. In general wrapper packages should be on the way out in lieu of directly using NPM packages.

You can either:

1. Just publish 2.1.0_9 and people on older versions won't get it, due to `versionsFrom('1.3')` (although it'll still show up on atmosphere etc).

2. Publish 2.1.1-rc.1 (which is not correct, I know).

**Comment by jsep** [comment]
@tmeasday  for now we just published under 2.1.1-rc.1

**Comment by hwillson** [comment]
As mentioned in https://github.com/meteor/meteor/issues/6499#issuecomment-197070764, wrapper packages are on the way out (they have been for a while now). Closing - thanks!
